wdl-engine/src/outputs.rs

```diff
@@ -65,28 +65,12 @@ impl Serialize for Outputs {
     {
         use serde::ser::SerializeMap;
 
-        /// Helper `Serialize` implementation for serializing element values.
-        struct Serialize<'a> {
-            /// The value being serialized.
-            value: &'a Value,
-        }
-
-        impl serde::Serialize for Serialize<'_> {
-            fn serialize<S>(&self, serializer: S) -> std::result::Result<S::Ok, S::Error>
-            where
-                S: serde::Serializer,
-            {
-                self.value.serialize(serializer)
-            }
-        }
```

Comment:
@peterhuene - I'm not certain I fully understand what this was doing before.

Reply:
I think this might have been a hold over from a time when `Value` wasn't directly serializable; it does seem entirely unnecessary.